client/src/components/calender.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import BigCalendar from 'react-big-calendar';
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+import EventForm from './eventForm.jsx'
+var moment = require('moment');
+
+BigCalendar.momentLocalizer(moment);
+moment().format();
```

Comment:
In the docs for Big-Calender it requires a date internalizer. I have just not begun utilizing it yet because I am still working on the click events.